archivebox/index/html.py

```diff
@@ -161,4 +161,4 @@ def snapshot_icons(snapshot) -> str:
             output += '<a href="{}" class="exists-{}" title="{}">{}</a> '.format(canon["archive_org_path"], str(exists),
                                                                                         "archive_org", icons.get("archive_org", "?"))
 
-    return format_html(f'<span class="files-icons" style="font-size: 1.1em; opacity: 0.8">{output}<span>')
+    return format_html('<span class="files-icons" style="font-size: 1.1em; opacity: 0.8">{}<span>', mark_safe(output))
```

Comment:
I think doing `mark_safe(output)` at any point would be dangerous because it allows untrusted text to slip by the XSS escaping process. (`format_html` is used to compose html fragments while still escaping them to avoid XSS) If an attacker put `'><script>onload('haxxed')</script` in their url's filename, it might render here and pwn our user.

I think providing them as separate args is enough 🤞, does this work instead?
```suggestion
    return format_html('<span class="files-icons" style="font-size: 1.1em; opacity: 0.8">{}<span>', output)
```

Reply:
Yep. Confirmed it still works for me.